src/Components/Header.js

```diff
@@ -0,0 +1,31 @@
+import { NavLink } from 'react-router-dom';
+import img from '../images/planet.png';
+
+const Header = () => (
+  <div>
+    <header className="header py-4 px-24 shadow-xl flex items-center justify-between">
+      <NavLink
+        to="/"
+        className={(isActive) => `${!isActive ? ' unselected' : ''} hidden sm:flex items-center gap-3`}
+      >
```

Comment:
- [ ] It is required that the active page should be indicated with an underline. Currently, your active page has a different color from the other pages which is a good implementation, but please let's indicate the active page with an underline as it is a requirement 😄 .